src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -89,28 +88,23 @@
         buildPemFileDialog().choose().ifPresent(file -> setPemFile(Paths.get(file.toURI())))
     );
     pemFileClearButton.addActionListener(__ -> clearPemFile());
+
     startDatePicker.addDateChangeListener(event -> {
-      if (!isDateRangeValid()) {
-        showError(MessageStrings.INVALID_DATE_RANGE_MESSAGE, "Export configuration error");
-        startDatePicker.clear();
-      } else
-        onSelectStartDateCallbacks.forEach(consumer -> consumer.accept(extractDate(event)));
+
+      if (event.getNewDate() != null)
+        endDatePicker.getSettings().setDateRangeLimits(event.getNewDate().plusDays(1), null);
```

Comment:
This block should be the same as on the `endDatePicker`:
- `... setDateRangeLimits(event.getNewDate(), null)`
- No need for the `if/else` case, no need for adding 1 day to the date